ui/src/app/job-details/panels/panels.component.html

```diff
@@ -19,7 +19,7 @@
       <p id="submitted" *ngIf="job.submission">Submitted {{ job.submission | jmShortDateTime }}</p>
       <p id="started" *ngIf="job.start">
         Started {{ job.start | jmShortDateTime }}
-        ({{ job.submission | jmDuration: job.end }})
+        ({{ job.start | jmDuration: job.end }})
```

Comment:
Not related to this PR: it seems the `job.status` and `{{ numCompletedTasks }} of {{ numTasks }} tasks completed` are in the same line now in the details page. I'm not sure is this what we want?

Reply:
Good catch @rexwangcc. I think they should be on separate lines or there should be some separation between the two, e.g. "Succeeded: 7 out of 7 tasks completed"